lib/utils.js

```diff
@@ -60,12 +58,21 @@ exports.domEach = function(cheerio, fn) {
  * string and then parsing the resultant markup.
  *
  * @argument {Object} dom - The htmlparser2-compliant DOM structure
- * @argument {Object} options - The parsing/rendering options
  */
-exports.cloneDom = function(dom, options) {
-  options = assign({}, options, { _useHtmlParser2: true });
+exports.cloneDom = function(dom) {
+  function filterOutParent(node) {
+    if (
+      "length" in dom
+        ? Array.prototype.some.call(dom, function(el) {
+          return node === el.parent;
+        })
+        : node === dom.parent
+    ) {
+      return null;
```

Comment:
This is tough to read syntactically, and the meaning is also a little hard to understand. What do you think about this:

```diff
+var parents = "length" in dom ?
+  Array.prototype.map.call(dom, function(el) { return el.parent; }) :
+  [dom.parent];
 function filterOutParent(node) {
-  if (
-    "length" in dom
-      ? Array.prototype.some.call(dom, function(el) {
-        return node === el.parent;
-      })
-      : node === dom.parent
-  ) {
+  if (parents.indexOf(node) > -1) {
     return null;
   }
 }
```

I have a feeling that `indexOf` will perform slightly better than `some`, but I'd recommend this even without that consideration.